tests/dialects/test_presto.py

```diff
@@ -193,6 +193,7 @@ def test_time(self):
             write={
                 "spark": "DAYOFWEEK(CAST('2012-08-08 01:00' AS TIMESTAMP))",
                 "presto": "DAY_OF_WEEK(CAST('2012-08-08 01:00' AS TIMESTAMP))",
+                "duckdb": "DAYOFWEEK(CAST('2012-08-08 01:00' AS TIMESTAMP))"
```

Comment:
Looks like this cast is invalid in duckdb:

```python
>>> duckdb.connect().execute("SELECT DAYOFWEEK(CAST('2012-08-08 01:00' AS TIMESTAMP))").fetchdf()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
duckdb.ConversionException: Conversion Error: timestamp field value out of range: "2012-08-08 01:00", expected format is (YYYY-MM-DD HH:MM:SS[.US][±HH:MM| ZONE])
```

Let's change it to a valid one: `DAYOFWEEK(CAST('2012-08-08 01:00:00' AS TIMESTAMP))`.

Reply:
Nice one, thanks for the heads up! Will make that change too

Reply:
Is this an entirely separate issue in that `timestamp '2012-08-08 01:00'` transpiles to `CAST('2012-08-08 01:00' AS TIMESTAMP)` in `duckdb` when it should be `CAST('2012-08-08 01:00:00' AS TIMESTAMP)`? Maybe I can add the test elsewhere to give something like `DAY_OF_WEEK('2012-08-08')` -> `DAYOFWEEK('2012-08-08')`?

Reply:
I meant that we can just add `:00` for this test. We don't need to transpile this case correctly for DuckDB.

Reply:
Cool, I've made changes now and I think it aligns with what you've said above but let me know if not!

Reply:
LGTM, thanks for the PR!